### DISCUSSION TITLE
Correct CR Touch Firmware for Ender 3 V2 with 4.3.1 board

### DISCUSSION THREAD

**Discussion by Waelve** [mriscoc/Ender3V2S1#1253] [Q&A]
I have an Ender 3 V2 3D Printer with a Direct Extruder, a CR Touch and a Double Z Axis, surfing the Internet I found this Repository where you can find the correct Firmware to use the CR Touch, however, when installing these files in the Printer ( Ender3V2-422-BLTUBL-MPC-20240125.bin & Ender3V2-422-MM-MPC-20240125. bin ), and go to the Info Screen Section it shows that the Size is 230x230x250 when in fact the print bed size is 220x220x250, I clarify, I already installed the two files and both show the same in Info Screen, so, my question is, which is the correct file/firmware to be able to update?
I attach images

![01-InfoScreen](https://github.com/mriscoc/Ender3V2S1/assets/24664763/206c6e37-4986-436f-a2d9-c4c4186c2266)
![02-MainMenu](https://github.com/mriscoc/Ender3V2S1/assets/24664763/d35a18e1-8ccd-4962-98b3-38f0675defe7)
![03-Impresora](https://github.com/mriscoc/Ender3V2S1/assets/24664763/3426c0e6-5bb8-4cb7-8865-b4aab1d91fc2)

**Comment by Crazy-Charles** [comment]
Confused... title says you have a 4.3.1 board (normally found in the Ender 6), you mention you tried 2 x 4.2.2 board firmware versions and the picture shows firmware for the 4.2.7 board!
Firstly make sure you load the correct firmware for the actual board you have if one is available.
Secondly - https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides to set bed sizes etc

**Comment by mriscoc** [comment]
Please check the firmware documentation: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
